.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
we show still allow this

Reply:
do we ? We have a minimal template so should be fine. This is just to make sure we have relevant information when bug is reported.

Reply:
blank for when people (or us) don't want to create a bug report. So either support a blank issue, or create a second template for "unsorted" which would be a single textarea with no requirements.

Reply:
good point. Enabled blank issue